slides/k8s/local-persistent-volumes.md

```diff
@@ -0,0 +1,242 @@
+# Local Persistent Volumes
+
+- We want to run that Consul cluster *and* actually persist data
+
+- But we don't have a distributed storage system
+
+- We are going to use local volumes instead
+
+  (similar conceptually to `hostPath` volumes)
+
+- We can use local volumes without installing extra plugins
+
+- However, they are tied to a node
+
+- If that node goes down, the volume becomes unavailable
+
+---
+
+## With or without dynamic provisioning
+
+- We will deploy a Consul cluster *with* persistence
+
+- That cluster's StatefulSet will create PVCs
+
+- These PVCs will remain unbound, until we will create local volumes manually
```

Comment:
I think we might need an explanation of what "unbound" means in this context.

Reply:
Right! It is explained in the previous chapters, but it doesn't hurt to remind it. So I added a footnote. Thanks!